client/my-sites/promote-post/main.tsx

```diff
@@ -141,9 +184,16 @@ export default function PromotedPosts( { tab }: Props ) {
 		);
 	}
 
-	const content = [ ...( posts || [] ), ...( pages || [] ), ...( products || [] ) ];
-
-	const isLoading = isLoadingPage && isLoadingPost && isLoadingProducts;
+	const content = [
+		...( mostPopularPostAndPages || [] ),
+		...( postAndPagesByComments || [] ),
+		...( products || [] ),
+	];
```

Comment:
Do we need to de-dupe these somehow? I managed to see the same post twice.

![de-dupping](https://user-images.githubusercontent.com/140841/221319103-c5d71793-5b8a-4599-be6e-7f988b04e998.png)

Reply:
> Do we need to de-dupe these somehow?

Good catch @DustyReagan, [Fixed it here](https://github.com/Automattic/wp-calypso/pull/73760/commits/8d184c2ed26828d644e69509f115c23854b5b1fa).

Reply:
Appears to deduping posts now! Nice!